core/src/test/java/org/wso2/testgrid/core/command/CleanUpCommandTest.java

```diff
@@ -23,19 +29,54 @@
 public class CleanUpCommandTest extends PowerMockTestCase {
```

Comment:
Good we are writing tests. What's the code coverage of this one?

Reply:
Method: 80%
Line 54%